karma.conf.js

```diff
@@ -24,7 +24,8 @@ module.exports = function(config) {
 			// source files, that you wanna generate coverage for
 			// do not include tests or libraries
 			// (these files will be instrumented by Istanbul)
-			'app/scripts/**/*.js': sourcePreprocessors
+			'app/scripts/**/*.js': sourcePreprocessors,
+			'test/spec/**/*.js': sourcePreprocessors
```

Comment:
Yes you could read it like that, but since it mostly talks about hard tested code and no example showed test code... well =)